camayoc/tests/qcs/cli/utils.py

```diff
@@ -109,9 +109,9 @@ def source_add(options, inputs=None, exitstatus=0):
     :param exitstatus: Expected exit status code.
     """
     if 'cred' in options:
-        options['cred'] = ' '.join(options['cred'])
+        options['cred'] = ''.join(string for string in options['cred'])
     if 'hosts' in options:
-        options['hosts'] = ' '.join(options['hosts'])
+        options['hosts'] = ''.join(string for string in options['hosts'])
```

Comment:
I'm not sure how this is preserving the space inbetween the items. Sure you have a scenario with multiple creds/hosts?

Reply:
You're right! I wasn't passing a single host/cred through in a list which was causing it to join them with a space in between the characters. I changed it back and passed them through in a list.